prov/tcp/src/tcpx.h

```diff
@@ -165,6 +165,12 @@ struct tcpx_pep {
 	struct poll_fd_info	poll_info;
 };
 
+enum tcpx_cm_state {
+	TCPX_EP_INIT,
+	TCPX_EP_CONN_ACTIVE,
+	TCPX_EP_CONN_SHUTDOWN,
+};
```

Comment:
We may want to have these states: CONNECTING, CONNECTED, SHUTDOWN, ERROR.  These are really active endpoint states, not necessarily CM states.  You can probably drop the 'CONN' from the name.

Reply:
updated